src/PHPUnit/Controller/AbstractControllerTestCase.php

```diff
@@ -226,14 +206,12 @@ public function getResponse()
 
     /**
      * Set the request URL
-     *
-     * @param  string                     $url
-     * @param  string|null                $method
-     * @param  array|null                 $params
-     * @return AbstractControllerTestCase
      */
-    public function url($url, $method = HttpRequest::METHOD_GET, $params = [])
-    {
+    public function url(
+        string $url,
+        ?string $method = HttpRequest::METHOD_GET,
+        ?array $params = []
+    ): AbstractControllerTestCase {
```

Comment:
Be aware that the same rules for `protected` BC breaks also apply to non-`final` `public` methods :|